include/irr/builtin/shaders/loaders/gltf/common.glsl

```diff
@@ -0,0 +1,11 @@
+#define _IRR_POSITION_ATTRIBUTE_ID = 0
+#define _IRR_NORMAL_ATTRIBUTE_ID = 1
+#define _IRR_UV_ATTRIBUTE_BEGINING_ID = 2
+#define _IRR_COLOR_ATTRIBUTE_BEGINING_ID = 5
+#define _IRR_JOINTS_ATTRIBUTE_BEGINING_ID = 8
+#define _IRR_WEIGHTS_ATTRIBUTE_BEGINING_ID = 12
+
+#define _IRR_MAX_UV_ATTRIBUTES = 3;
+#define _IRR_MAX_COLOR_ATTRIBUTES = 3;
+#define _IRR_MAX_JOINTS_ATTRIBUTES = 4;
+#define _IRR_MAX_WEIGHTS_ATTRIBUTES = 4;
```

Comment:
prefix with `_IRR_BUILTIN_GLSL_GLTF_` instead

Reply:
also preprocessor defines don't take a `=` nor a `;`

Reply:
oh right

Reply:
hangover did the job XD

Reply:
corrected in following commit